features/graph/api/pom.xml

```diff
@@ -32,6 +32,10 @@
             <artifactId>org.opennms.features.graph.jung</artifactId>
             <version>${jungVersion}</version>
         </dependency>
+        <dependency>
+            <groupId>org.opennms</groupId>
+            <artifactId>opennms-model</artifactId>
```

Comment:
This is a pretty heavy dependency to add to the API module :)

Looks like there's no easy way around it if we want to use the `OnmsSeverity` model though.